profile/profile.yaml

```diff
@@ -0,0 +1,99 @@
+# Preferred name (Characters only)
+name: "Bhavya Vinod"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_BhavyaVinod
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: I am a results-oriented person who is passionate about software development. I have more than two years of experience as an IT professional. This experience helped me to build a detail-oriented, analytical mindset and exposed me to the work culture and the agile methodologies followed in the large-scale industries. Before pursuing a career in Data Science, I completed my degree in Data analytics for business. This program covered Machine learning, Data visualization, Financial Analytics, and Advanced statistics. Through my passion to understand the needs of clients and the ability to quickly learn new technologies, I will be a great fit to the team. 
```

Comment:
summary: I am a results-oriented person who is passionate about software development. I have more than two years of experience as an IT professional. This experience helped me build a detail-oriented, analytical mindset and exposed me to the work culture and the agile methodologies followed in large-scale industries. Before pursuing a career in Data Science, I completed my degree in Data analytics for business. This program covered Machine learning, Data visualization, Financial Analytics, and Advanced statistics. Through my passion for understanding the needs of clients and the ability to learn new technologies quickly, I will be a great fit for the team. 

fixed grammar